### PR TITLE
Add skip link component and main container to admin layout

### PR DESCRIPTION
**Pull request by alex-ju**
This PR:
- adds skip link component based on GOVUK Frontend
- fix: moves the header and footer from the admin layout published in gem to the dummy admin layout in spec
- adds container wrapper and main element to the admin layout
- updates changelog

---

Component guide for this PR:
https://govuk-publishing-compon-pr-417.herokuapp.com/admin

### CODE REVIEW

File: app/views/govuk_publishing_components/components/docs/skip_link.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Skip link
+description: Skip link component helps keyboard-only users skip to the main content on a page
+accessibility_criteria: |
+  - the link is the first items that screen readers hear and that keyboard users tab to
+  - is becomes visible when pressing tab
```
Comment (maxgds) [inline app/views/govuk_publishing_components/components/docs/skip_link.yml]: typo, remove "is"

File: app/views/govuk_publishing_components/components/docs/skip_link.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+shared_accessibility_criteria:
+- link
```
Comment (maxgds) [inline app/views/govuk_publishing_components/components/docs/skip_link.yml]: Not all link criteria are relevant, (the link doesn't have visible text, for example, until it's in focus) and the component IS a link, rather than a container for links. I'd pick and choose the relevant ones and put them in criteria and leave out the shared ones.

File: spec/dummy/app/views/welcome/admin_example.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<%= render 'govuk_publishing_components/components/title', title: "Hello! I am an example admin page" %>
+<h1 class="govuk-heading-xl">Hello! I am an example admin page</h1>
```
Comment (maxgds) [inline spec/dummy/app/views/welcome/admin_example.html.erb]: Why this change? Isn't this is what the title component is for? If you need to add a class and the component doesn't allow, we should probably extend it rather than not use it. Or was it rendering a h2 or something previously?

File: app/views/govuk_publishing_components/components/docs/skip_link.yml (hunk 2 above)
Comment (alex-ju) [reply app/views/govuk_publishing_components/components/docs/skip_link.yml]: Updated as suggested (dropped the shared criteria an rewrite explicitly

File: spec/dummy/app/views/welcome/admin_example.html.erb (hunk 3 above)
Comment (alex-ju) [reply spec/dummy/app/views/welcome/admin_example.html.erb]: Since we're trying to use the GOVUK Frontend components (which, for example, only have margins at the bottom) and layout for the admin interface I switched to using a simple heading instead of a title component

Comment (maxgds) [reply spec/dummy/app/views/welcome/admin_example.html.erb]: Ok, I think that makes sense. I'm used to using the existing gem components everywhere so maybe need to get out of that mindset.

Comment (alex-ju) [reply spec/dummy/app/views/welcome/admin_example.html.erb]: I agree, I intend to do the same (use gem components everywhere possible), but in this case the title component brings more than I need to the example.

Comment (nickcolley) [reply spec/dummy/app/views/welcome/admin_example.html.erb]: Perhaps the 'core' (https://github.com/alphagov/govuk-frontend/tree/master/src/core) parts of GOV.UK Frontend do not need to be wrapped but any 'components' should.

One of the best bits of having everything wrapped as components is if we need to change any markup (for example the class names change) it can be done in one place.
